src/sites/dMCDA.jl

```diff
@@ -182,12 +205,47 @@ function rank_sites!(S, weights, rankings, n_site_int, mcda_func, rank_col)::Tup
     return prefsites, s_order
 end
 
-function rank_seed_sites!(S, weights, rankings, n_site_int, mcda_func)::Tuple{Vector{Int64},Matrix{Union{Float64,Int64}}}
-    rank_sites!(S, weights, rankings, n_site_int, mcda_func, 2)
+"""
+    retrieve_ranks(S::Matrix, weights::Vector{Float64}, mcda_func::Function, site_ids::Vector)
+    retrieve_ranks(S::Matrix, weights::Vector{Float64}, mcda_func::Vector{Any}, site_ids::Vector)
+    retrieve_ranks(S::Matrix, scores::Vector, rev_val::Bool, site_ids::Vector)
+
+Get location ranks using mcda technique specified in mcda_func, weights and a decision matrix S.
+
+# Arguments
+- `S` : decision matrix containing criteria values for each location (n locations)*(m criteria)
+- `weights` : importance weights for each criteria. 
+- `mcda_func` : function/[function bool] array to use for mcda, specified as an element from method.
+- `site_ids` : array of site ids still remaining after filtering.
+- `scores` : set of scores derived from applying an mcda ranking method.
```

Comment:
Orphaned arguments in argument list?

Reply:
which one?

Reply:
Read the argument list and then compare with the function signatures given at the top, and the function signature that is implemented.

Reply:
List of functions in docstring still doesn't match implementations?

Reply:
@Rosejoycrocker I guess you did update them but they're still incorrect.

Corrected, and changed `rev_val` to `maximize` as the latter is more intuitive.